recipes/odm2rest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0a" %}
+
+package:
+  name: odm2rest
```

Comment:
@lsetiawan I'm assuming this the package name. ODM2 is quite confusing b/c the repo name is different from the package name that is different from the import name...

Reply:
That is the package name.

> ODM2 org is quite confusing b/c the repo name is different from the package name that is different from the import name...

Import and package name should be the same for this repo, I gotta change the repo name 😛

Reply:
>> ODM2 org is quite confusing b/c the repo name is different from the package name that is different from the import name...
>
> Import and package name should be the same for this repo, I gotta change the repo name 

Like @lsetiawan said, the import and package names are the same for this repo, and that's what we're shooting for in all cases in ODM2. The repo name is legacy. I'm reluctant to change it unilaterally w/o consulting others first. But having a repo name that's different from the import/package name is not extremely unusual, I think.